foundry.toml

```diff
@@ -1,2 +1,3 @@
 [fuzz]
 runs = 10000
+max_test_rejects = 1000000
```

Comment:
This seems like a high value. Do you know if it's mainly due to [`vm.assume(false)`](https://github.com/a16z/erc4626-tests/blob/8b1d7c2ac248c33c3506b1bff8321758943c5e11/ERC4626.prop.sol#L373)?

Reply:
Yes, that's my understanding based on the document: https://book.getfoundry.sh/cheatcodes/assume

Indeed, it can be reduced to 100K, if you prefer it to be tighter. Please let me know!

Reply:
If the default is 65536, bumping it to 100k doesn't sound like that big of a deal. I'm still wondering if there's a better way to implement this that doesn't rely on `vm.assume(false)` so much. It would need to be a way of avoiding triggering reverts, instead of catching them and discarding the run post-facto.

Reply:
Since the tests are written to be agnostic to the implementation details (so that they are applicable to various implementations), it is hard to precisely find the input range that never reverts for various conditions.

An option is to bound inputs for, e.g., withdraw() using previewWithdraw() or convertToAssets(), but as you know, the preview or convert functions do not need to be precise and they may be conservative, so relying on them may exclude some edge cases. (Indeed, setting `_unlimitedAmount` flag to false will exactly do this.)

I haven't found a better way to not exclude edge cases without the trial-and-error approach, but please let me know if we have an idea!

Reply:
I see, that makes a lot of sense. Let's bump to 100k.

Reply:
Yep, now it's 100K.